lib/compat/wordpress-6.0/get-global-styles-and-settings.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * API to interact with global settings & styles.
+ *
+ * @package gutenberg
+ */
+
+ if ( ! function_exists( 'wp_get_global_styles_svg_filters' ) ) {
+	/**
+	 * Returns a string containing the SVGs to be referenced as filters (duotone).
+	 *
+	 * @return string
+	 */
+	function wp_get_global_styles_svg_filters() {
```

Comment:
I see that the `wp_get_global_styles_svg_filters` is not present in WordPress 5.9 but still lives in the `lib/compat/5.9` folder of Gutenberg. Apparently, https://github.com/WordPress/gutenberg/pull/36236 didn't make the cut for 5.9 and is slated for the next minor release (it has the "backport to WP minor release" label).

My understanding of how we use the `lib/compat` folder is that we should move the existing `wp_get_global_styles_svg_filters` to a `lib/compat/5.9.1` folder. Once there, we can make the changes introduced by this PR directly, as it wasn't shipped in a WordPress version yet.

cc @gziolo and @youknowriad for confirmation.

Reply:
I'm not sure if we need to be so much specific about the minor version because we remove code in batch for a major version line. The existing `lib/compat/5.9` directory should be fine.

Reply:
So are you saying we didn't need to revert https://github.com/WordPress/gutenberg/pull/38055 ?

Reply:
Ah, I see. For efficiency, I asked for further clarifications in private about what happens with PRs that need to go on a minor and this is what I've learned:

- they should be labeled with "[Backport to WP minor](https://github.com/WordPress/gutenberg/labels/Backport%20to%20WP%20Minor%20Release)"
- they go in the folder of the major version, in this case, `lib/compat/5.9`

We did use a 5.8.1 folder in the past but it's not clear that was super useful.

There's the issue of what happens with code that was supposed to go in a minor but a minor didn't happen: it seems that we need to port that code to the folder of the next major (`lib/compat/6.0` in this case) at some point (I presume we do this review when we try to remove the `compat` folder, but I'm not sure how/when this happens, to be honest).

---

So, the way forward to incorporate this feature would be:

- make sure the dependency is marked for a minor https://github.com/WordPress/gutenberg/pull/36236 (done)
- go with the 1st attempt https://github.com/WordPress/gutenberg/pull/38055 and label it for minor as well

Appreciate your patience here, @scruffian and @ajlende Sorry to have you go round in circles.